src/lessons/Data-Structures/03.md

```diff
@@ -0,0 +1,56 @@
+As we've seen, the centralized web relies on central authorities
+to host our data and uses URLs as addresses to access it. But location
+addressing isn't the only option. On the
+<strong>decentralized web</strong>, we can all host each other's data,
+and there's a different kind of linking that's more secure, making it easy to
+trust our neighbors.
+
+## Crytographic hashing
+
+<strong>[Cryptographic hashing](https://en.wikipedia.org/wiki/Cryptographic_hash_function)</strong> is the
+most important tool in the toolbox of decentralized data structures. It opens the door to a new form of linking, known as <strong>content
+addressing</strong>, that we can use in a decentralized environment without central authorities.
+
+Hashing takes data of any size and returns you a single fixed-size "hash" for that data.
+A hash is a string of characters that looks like gobbledygook, but you can think of it
+as a link. It might look something like this:
```

Comment:
"think of it as a link" feels like we skipped a step. "think of it as a name" seems to be in-line with where the current narrative is at. Somewhere towards the end we can note "if we use these names as links..."